.github/actions/spelling/line_forbidden.patterns

```diff
@@ -1,119 +1,289 @@
 #\sin to\s(?!if\b)
 
-# s.b. is obsolete
+# Should be `use`
+\sin used by\b
+
+# Should be `is obsolete`
 \bis obsolescent\b
 
-# s.b. it's or its
+# Should be `it's` or `its`
 \bits['’]
 
-# s.b. opt-in
-#(?<!\sfor)\sopt in\s
+# Should be `its`
+\bit's(?= own\b)
+
+# Should be `log in`
+\blogin to the
+
+# Should be `long-standing`
+\blong standing\b
+
+# Should probably be a person named `Nick` or the abbreviation `NIC`
+\bNic\b
+
+# Should be `not supposed`
+\bsupposed not\b
 
-# s.b. less than
+# Should probably be `much more`
+\bmore much\b
+
+# Should be `perform its`
+\bperform it's\b
+
+# Should be `opt-in`
+#(?<!\scan|for)(?<!\sif)\sopt in\s
+
+# Should be `less than`
 \bless then\b
 
-# s.b. one of
-\bon of\b
+# Should be `load balancer`
+\b[Ll]oud balancer
+
+# Should be `moot`
+\bmute point\b
+
+# Should be `one of`
+(?<!-)\bon of\b
+
+# Should be `on the other hand`
+\b(?i)on another hand\b
+
+# Most people only have two hands. Reword.
+\b(?i)on the third hand\b
 
-# s.b. otherwise
+# Should be `otherwise`
 \bother[- ]wise\b
 
-# s.b. or (more|less)
+# Should be `or (more|less)`
 \bore (?:more|less)\b
 
-# s.b. nonexistent
+# Should be `rather than`
+\brather then\b
+
+# Should be `regardless, ...` or `regardless of (whether)`
+\b[Rr]egardless if you\b
+
+# Should be `no longer needed`
+\bno more needed\b(?! than\b)
+
+# Should be `did not exist`
+\bwere not existent\b
+
+# Should be `nonexistent`
 \bnon existing\b
+
+# Should be `nonexistent`
 \b[Nn]o[nt][- ]existent\b
 
-# s.b. brief / details/ param / return / retval
+# Should be `@brief` / `@details` / `@param` / `@return` / `@retval`
 (?:^\s*|(?:\*|//|/*)\s+`)[\\@](?:breif|(?:detail|detials)|(?:params(?!\.)|prama?)|ret(?:uns?)|retvl)\b
 
-# s.b. preexisting
+# Should be `preexisting`
 [Pp]re[- ]existing
 
-# s.b. preempt
+# Should be `preempt`
 [Pp]re[- ]empt\b
 
-# s.b. preemptively
+# Should be `preemptively`
 [Pp]re[- ]emptively
 
-# s.b. recently changed or recent changes
+# Should be `prepopulate`
+[Pp]re[- ]populate
+
+# Should be `prerequisite`
+[Pp]re[- ]requisite
+
+# Should be `recently changed` or `recent changes`
 [Rr]ecent changed
 
-# s.b. reentrancy
+# Should be `reentrancy`
 [Rr]e[- ]entrancy
 
-# s.b. reentrant
+# Should be `reentrant`
 [Rr]e[- ]entrant
 
-# s.b. understand
+# Should be `strong suit`
+\b(?:my|his|her|their) strong suite\b
+
+# Should be `understand`
 \bunder stand\b
 
-# s.b. workarounds
+# Should be `URI` or `uri` unless it refers to a person named `Uri`
+#(?<!\.)\bUri\b(?![(])
+
+# Should be `it uses is`
+/\bis uses is\b/
+
+# Should be `uses it as`
+(?:^|\. |and )uses is as (?!an?\b|follows|livestock|[^.]+\s+as\b)
+
+# Should be `was`
+\bhas been(?= removed in v?\d)
+
+# Should be `where`
+\bwere they are\b
+
+# should be `vCenter`
+\bV[Cc]enter\b
+
+# Should be `VM`
+\bVm\b
+
+# Should be `workarounds`
 #\bwork[- ]arounds\b
 
-# s.b. workaround
+# Should be `workaround`
 (?:(?:[Aa]|[Tt]he|ugly)\swork[- ]around\b|\swork[- ]around\s+for)
 
```

Comment:
MS isn't alone:
https://github.com/search?q=%2F%5Cb%5BWw%5Dalk%5B-+%5Daround+for%5Cb%2F+NOT+%2F%5Cb%5BWw%5Dalk%5B-+%5Daround+for+%28%3F%3A%5Cd%2B%7Can%3F%29%5Cb%2F&type=code

I think I'll try using it for a bit and see how often I hit the positives.

You're right that quite a few of the instances in Microsoft are bugs:
https://github.com/search?q=%2F%5Cb%5BWw%5Dalk%5B-+%5Daround%5Cb%2F+org%3Amicrosoft&type=code&p=2
(Although on page 1, a number of them were correct...)

I might try writing a fancier rule for spell-check-this since I think it's probably possible to more precisely characterize the error cases. But not tonight.